src/datta_lab_to_nwb/markowitz_gillis_nature_2023/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .markowitz_gillis_nature_2023behaviorinterface import MarkowitzGillisNature2023BehaviorInterface
-from .fiberphotometryinterface import FiberPhotometryInterface
-from .markowitz_gillis_nature_2023nwbconverter import MarkowitzGillisNature2023NWBConverter
+from datta_lab_to_nwb.markowitz_gillis_nature_2023.behaviorinterface import BehaviorInterface
+from datta_lab_to_nwb.markowitz_gillis_nature_2023.fiberphotometryinterface import FiberPhotometryInterface
+from datta_lab_to_nwb.markowitz_gillis_nature_2023.nwbconverter import NWBConverter
```

Comment:
Please use relative imports inside `src`

Reply:
yes, see [coding style guide](https://neuroconv.readthedocs.io/en/main/developer_guide/style_guide.html#scikit-learn-style). @CodyCBakerPhD and @h-mayorquin , maybe it makes sense to move the style guide somewhere that is organization-wide

Reply:
There is the hidden repo https://github.com/catalystneuro/.github

I don't know how much of a stand-alone repo is truly is (like, can we generate a ReadTheDocs for all CatalystNeuro there?)

But that's the top-most level I can think; at a minimum maybe add a line to the README if that's not too out of place

Reply:
Happy to follow the style guide, just curious though, is there a rationale for using relative imports inside `src`?

Reply:
> is there a rationale for using relative imports inside src?

Yes, and it's been discussed extensively across various issues/PRs on various repos which led to the documentation of that practice in the coding style

I can try to dig up the links for them if you want but the short summary is (a) absolute paths are for testing b/c that's how a user would actually call it after `pip` installation, (b) pure relatives don't even need a local install to import, (c) absolutes in `src` are not as persistent to relative changes (if you restructure the folders of a bigger package), (d) it's also just less typing when all is said and done, which given the nature of this PR I figured you'll also be all-for lol

Reply:
Cool, good enough for me!